python-sdk/nuscenes/eval/predict/splits.py

```diff
@@ -0,0 +1,37 @@
+# nuScenes dev-kit.
+# Code written by Freddy Boulton.
```

Comment:
Since I cannot comment on it directly: The `prediction_scenes.json` file makes up 2/3 of the nuscenes-devkit repo size. It would be better to remove it from Git.

Reply:
Ok I will add a dataroot parameter to this function. The understanding is that we'll release the `prediction_scenes.json` as part of the nuscenes dataset